salt/cloud/clouds/dimensiondata.py

```diff
@@ -38,6 +38,7 @@
 
     HAS_LIBCLOUD = True
 except ImportError:
+    log.exception("You do not have apache-libcloud 0.20<= installed")
```

Comment:
can https://github.com/saltstack/salt/blob/develop/salt/cloud/libcloudfuncs.py#L51 LIBCLOUD_MINIMAL_VERSION be specified per module?